### PR TITLE
patching aten failures on python tests

### PR DESCRIPTION
**Pull request by jjsjann123**
Fixes failing test_binary_ops on `pow` by avoiding failing config

### CODE REVIEW

File: test/test_jit_cuda_fuser.py [hunk 1]
```diff
@@ -786,6 +786,18 @@ def t(x: torch.Tensor, y: torch.Tensor, z: torch.Tensor):
 
         z = torch.tensor([2], device="cuda").to(dtype_arg1)
 
+        if operation in [torch.pow]:
+            if dtype_arg1 == torch.int32 or dtype_arg2 == torch.int64:
```
Comment (zasdfgbnm) [inline test/test_jit_cuda_fuser.py]: Does this test works for `torch.pow(int64_tensor, int32_tensor)`?

**Review by zasdfgbnm — APPROVED** [review]

File: test/test_jit_cuda_fuser.py (hunk 1 above)
Comment (jjsjann123) [reply test/test_jit_cuda_fuser.py]: Oops, good catch. This should have been disabled for when both inputs are integers. Let me update that.

**Comment by csarofeen** [comment]
Is this related to https://github.com/pytorch/pytorch/issues/73196 ? If we fix that issue should we revert this PR or parts of it?

**Comment by csarofeen** [comment]
Is https://github.com/csarofeen/pytorch/pull/1479 related to this?

**Comment by jjsjann123** [comment]
Forgot that I opened the other issue. :)

1. yes, this is caused by pytorch#73196, if we fix that we would want to revert this one.
2. no #1479 fixes errors in nvfuser for integral/boolean input tensors, the failure we are patching here comes from aten.

**Pull request merged**